library/rssbuilder/doc/changelog_rssbuilder.htm

```diff
@@ -26,7 +26,7 @@ <h3>2.2 <small>(2005-12-04)</small></h3>
 <h3>2.1 <small>(2004-03-25)</small></h3>
 <ul>
 	<li>Updated code for PHP&nbsp;5 RC1. Will not work under PHP&nbsp;4!</li>
-	<li>Completly rewritten classes which take use of PHP5's new DOM functions!</li>
+	<li>Completely rewritten classes which take use of PHP5's new DOM functions!</li>
```

Comment:
Remove this change, there is no point in patching comments in a 3rd party library